### PR TITLE
Execution tab when RBE enabled

### PR DESCRIPTION
**Pull request by pariparajuli**
Tab for executions available, accessible page during dense mode

### CODE REVIEW

**Comment by pariparajuli** [comment]
Screen Shot 2021-08-17 at 12 06 37 PM
Screen Shot 2021-08-17 at 12 14 33 PM

File: app/invocation/invocation.tsx [hunk 1]
```diff
@@ -286,9 +291,7 @@ export default class InvocationComponent extends React.Component<Props, State> {
             />
           )}
 
-          {activeTab == "timing" && <TimingCardComponent model={this.state.model} />}
-
-          {activeTab == "execution" && (
+          {isBazelInvocation && (activeTab == "all" || activeTab == "execution") && (
```
Comment (siggisim) [inline app/invocation/invocation.tsx:294]: I don't think we want the executions tab to show up on the all page

**Review by siggisim — CHANGES_REQUESTED** [review]
Looks good, just a minor comment

File: app/invocation/invocation_execution_card.tsx [hunk 2]
```diff
@@ -275,12 +274,8 @@ export default class ExecutionCardComponent extends React.Component {
 
     return (
       <div>
-        <InvocationFilterComponent
-          hash={"#execution"}
-          search={this.props.search}
-          placeholder={"Filter by digest or command..."}
-        />
         <div className={`card expanded`}>
+          <img className="icon" src="/image/cloud-regular.svg" />
```
Comment (siggisim) [inline app/invocation/invocation_execution_card.tsx:278]: I'm not seeing this icon in the screenshots - where does this show up?

**Review by siggisim — CHANGES_REQUESTED** [review]

File: app/invocation/invocation_execution_card.tsx (hunk 2 above)
Comment (pariparajuli) [reply app/invocation/invocation_execution_card.tsx:278]: Realized that the screenshots are from before i added the icon but this is what it looks like after 
Screen Shot 2021-08-19 at 4 20 36 PM

Comment (siggisim) [reply app/invocation/invocation_execution_card.tsx:278]: Could we maybe just leave it out? It pushes all of the content to the right.

**Review by siggisim — APPROVED** [review]
Looks great!

**Pull request merged**